edx_proctoring/tests/test_utils.py

```diff
@@ -66,3 +70,18 @@ def test_emit_event(self):
                 'one': 'two'
             }
         )
+
+    def test_has_client_app_shutdown(self):
```

Comment:
I don't get where this test is coming from and for, given that we are removing polling?  Was this moved, but I'm not seeing the original?  Other?